_posts/2019-11-27-gettingstarted-2020/getting-started-2020.Rmd

```diff
@@ -0,0 +1,144 @@
+
+Overall, the structure of what follows is this. We start from the core question: _How do you build a model?_, then frame it from both sides; i.e.: _What comes before?_ (data loading / preprocessing) resp. _What comes after?_ (model saving / deployment).
+
+After that,  we quickly go into creating models for different types of data: images, text, tabular. 
+
+Then, we touch on where to find background information, such as: How do I add a custom callback? How do I create a custom layer? How can I define my own training loop?
+
+Finally, we round up with something that looks like a tiny technical addition but has far greater impact: integrating modules from TensorFlow (TF) Hub.
+
+
+## Getting started
+
+### How to build a model?
+
+If linear regression is the Hello World of machine learning, non-linear regression has to be the Hello World of neural networks. The [Basic Regression tutorial](https://tensorflow.rstudio.com/tutorials/beginners/basic-ml/tutorial_basic_regression/) shows how to train a dense network on the Boston Housing dataset. This example uses the Keras [Functional API](https://tensorflow.rstudio.com/guide/keras/functional_api/), one of the two "classical" model-building approaches -- the one that tends to be used when some sort of flexibility is required. In this case, the desire for flexibility comes from the use of [feature columns](https://tensorflow.rstudio.com/guide/tfdatasets/feature_columns/) -  a nice new addition to TensorFlow that allows for convenient integration of e.g. feature normalization (more about this in the next section).
+
+<aside>
+The [regression tutorial](https://tensorflow.rstudio.com/tutorials/beginners/basic-ml/tutorial_basic_regression/) now uses feature columns for convenient data preprocessing.
+</aside>
+
+This introduction to regression is complemented by a [tutorial on multi-class classification](https://tensorflow.rstudio.com/tutorials/beginners/basic-ml/tutorial_basic_classification/) using "Fashion MNIST". It is equally suited for a first encounter with Keras.
+
+A third tutorial to be mentioned in this section is dedicated to [text classification](https://tensorflow.rstudio.com/tutorials/beginners/basic-ml/tutorial_basic_text_classification/). Here too, there is a hidden gem in the current version that makes text preprocessing a lot easier: `layer_text_vectorization`, one of the brand new [Keras preprocessing layers](https://github.com/keras-team/governance/blob/master/rfcs/20190502-preprocessing-layers.md).^[In fact, it is that new that as of this writing, you will have to install the nightly build of TensorFlow -- as well as `tensorflow` from github -- to use it.] If you've used Keras for NLP before: No more messing with `text_tokenizer`!
```

Comment:
```suggestion
A third tutorial in this section is dedicated to [text classification](https://tensorflow.rstudio.com/tutorials/beginners/basic-ml/tutorial_basic_text_classification/). Here too, there is a hidden gem in the current version that makes text preprocessing a lot easier: `layer_text_vectorization`, one of the brand new [Keras preprocessing layers](https://github.com/keras-team/governance/blob/master/rfcs/20190502-preprocessing-layers.md).^[In fact, it is so new that as of this writing, you will have to install the nightly build of TensorFlow -- as well as `tensorflow` from github -- to use it.] If you've used Keras for NLP before: No more messing with `text_tokenizer`!
```

Reply:
I removed "to be mentioned" from the first sentence, since you're in the act of mentioning it already!

Reply:
Also changed "it is that new..." to "it is **so** new..."

Reply:
Right :-))